app/Jobs/SendTransfer.php

```diff
@@ -50,9 +50,11 @@ public function handle(NexmoServiceProvider $nexmo)
             Mail::to($email)->send(new NewPayment($this->transfer));
         }
 
+        $link = config('app.transfer_link') . $this->transfer->token;
```

Comment:
I'll name it `$transfer_link`, because it's not the token itself, how about that? 
> Not camel case following PHP code conventions.